ccd-definition/AuthorisationCaseField/CareSupervision/gatekeeper.json

```diff
@@ -2539,5 +2539,61 @@
     "CaseFieldID": "showFurtherEvidenceTab",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
+  },
```

Comment:
afaik gatekeeper role is removed from all users, do we even need to update this? in fact can we just delete the file (in a separate PR). guess idam would have be queried first

Reply:
ahh, forgot about that, was just keeping things consistent and was only testing with the gatekeeper-only user. I guess it would be a future clean up